dolt_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,267 @@
+```
+$ cd path/to/hospital-price-transparency
+$ dolt sql
+# Welcome to the DoltSQL shell.
+# Statements must be terminated with ';'.
+# "exit" or "quit" (or Ctrl-D) to exit.
+hospital_price_transparency> select commit_hash, message from dolt_commits where `date` < '2021-02-17' order by `date` desc limit 10;
++----------------------------------+------------------------------------------------------------+
+| commit_hash                      | message                                                    |
++----------------------------------+------------------------------------------------------------+
+| f0lecmblorr67rcuhuti6tbkriigh6gt | Updating prices with changes from uwmc_prices.csv          |
+| mj9ce6d8em9avj9ej0pqnaoes4fbglti | Updating cpt_hcpcs with changes from uwmc_cpt_hcpcs.csv    |
+| 2j6ommult20qvbj05j1nq63nkbd5fgdj | Updating prices with changes from prices.csv               |
+| pu8ctvhfcpp83q3iil8trp90vnuesaci | Updating cpt_hcpcs with changes from cpt_hcpcs.csv         |
+| q49l0kgnbbbgkt3imjd57tslbi2iges8 | Updating hospitals with changes from hospitals.csv         |
+| gstcq5loi9ieqdv1elrljab9hcgr090p | Updating hospitals with changes from hospitals.csv         |
+| te6spcqtjk0scose2c45f9t7tpcrt69c | Added hospital WellSpan Surgery & Rehabilitation Hospital. |
+| bjg3b5lua8omadcl5nr6o7v0nphliqpu | Added hospital WellSpan York Hospital.                     |
+| t4js1g5mfvgikqlmqa238it26mg94g5i | Added hospital Children's of Alabama.                      |
+| jsan7p4iad61cjmeti858ebcl4s86vda | Added hospital McLaren Lapeer Region.                      |
++----------------------------------+------------------------------------------------------------+
+```
+
+Suppose we'd like to run our pipeline with input data as of commit `gstcq5loi9ieqdv1elrljab9hcgr090p`. That's easy enough, first let's name the commit with a branch:
+```
+$ dolt branch metaflow-backtest gstcq5loi9ieqdv1elrljab9hcgr090p
+```
+
+Now let's kick off recomputing the medians, noting that we also wrote our medians to a new branch to ensure we didn't overwrite the HEAD value of our medians:
+```
+$ poetry run python3 hospital_procedure_price_state_medians.py run \ 
+    --hospital-price-db path/to/hospital-price-transparency \ 
+    --hospital-price-db-branch metaflow-backtest \
+    --hospital-price-analysis-db path/to/hospital-price-analysis
+    --hospital-price-analysis-db-branch metaflow-backtest
+```
+
+And the variances:
+```
+$ poetry run python3 hospital_procedure_price_variance_by_state.py run \ 
+    --hospital-price-analysis-db path/to/hospital-price-analysis
+    --hospital-price-analysis-db-branch metaflow-backtest
+```
+
+We can now query the results directly from the Metaflow client API alongside the integration:
+```python
+from metaflow import Flow
+from dolt_integrations.metaflow import DoltDT
+dolt = DoltDT(run=Flow('HospitalProcedurePriceVarianceByState').latest_successful_run)
+df = dolt.read('variance_by_procedure')
+print(df)
```

Comment:
can you show a head or something? for the `dolt sql` below also, missing output makes my lizard brain unhappy

Reply:
agreed